core/web-assets/src/main/assets/js/apps/onms-classifications/index.js

```diff
@@ -312,6 +314,59 @@ const confirmTopoverTemplate = require('./views/modals/popover.html');
                 });
             };
 
+            // for drag and drop of rules (redefining position)
+            $scope.sortableRules = {
+
+                start: function(e, ui) {
+                    // remember old index before moving
+                    $(ui.item).data('oldIndex', ui.item.index());
```

Comment:
Ideally `angular.element(ui.item)` would be used instead.

Reply:
fixed.